panther

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Psy\Shell;
+use Symfony\Component\Panther\Client;
+use Symfony\Component\Panther\PantherTestCaseTrait;
+
+require file_exists($autoload = __DIR__.'/vendor/autoload.php') ? $autoload : __DIR__.'/../../autoload.php';
+
+if ($psyshBin = $_SERVER['PANTHER_PSYSH_BIN'] ?? false) {
```

Comment:
Oh sorry I may have expressed myself poorly: it should still default to the recommended path:

```suggestion
$psyshPath = $_SERVER['PANTHER_PSYSH_BIN'] ?? '/usr/local/bin/psysh';
if (file_exists($$psyshPath)) {
    require $psyshBin;
```